firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/FeedbackActivity.java

```diff
@@ -67,29 +71,30 @@ private void setupView() {
     Button submitButton = this.findViewById(R.id.submitButton);
     submitButton.setOnClickListener(this::submitFeedback);
 
-    Bitmap thumbnail = readThumbnail();
-    if (thumbnail != null) {
+    try {
+      Bitmap thumbnail = readThumbnail();
       ImageView screenshotImageView = this.findViewById(R.id.thumbnail);
       screenshotImageView.setImageBitmap(thumbnail);
-    } else {
+    } catch (FirebaseAppDistributionException e) {
```

Comment:
Instead of using exceptions for flow control, you could have `readThumbnail()` return `null` instead of throwing an exception if there's no screenshot available.

But even simpler: Just check whether `screenshotUri` is null before trying to read the thumbnail